sln/WebApiOData.Performance.Official.sln

```diff
@@ -0,0 +1,38 @@
+﻿
```

Comment:
What's the difference between this and WebApiOData.Performance.sln ? Why two solutions for perf?